docs/docs/how-to/file-uploads.md

```diff
@@ -446,6 +446,50 @@ Now let's add the ability to bring back the uploader if you decide you want to c
 
 We're borrowing the styles from the submit button and making sure that the image has both a top and bottom margin so it doesn't crash into the new button.
 
+## Safe usage of your free plan
```

Comment:
I've got a theme running through this doc that every title starts with "The"! 😉 And all the headings are Title Case and have a newline between the heading and the following paragraph.

```suggestion
## The Delete

```